src/GUI/Overlay/OverlaySystem.cpp

```diff
@@ -37,13 +40,16 @@ OverlaySystem::~OverlaySystem() {
 }
 
 void OverlaySystem::addOverlay(std::string_view name, std::unique_ptr<Overlay> overlay) {
+    if (!_overlays.contains(name.data())) {
```

Comment:
You can't call `data()` on a `string_view` and expect it to work. Use `_overlays.contains(name)`.